cmd/agent/main.go

```diff
@@ -1,3 +1,40 @@
 package main
 
-func main() {}
+import (
+	"context"
+	"log"
+	"os"
+	"syscall"
+	"time"
+
+	"github.com/VadimOcLock/metrics-service/internal/config"
+
+	"github.com/VadimOcLock/metrics-service/internal/worker"
+	"github.com/VadimOcLock/metrics-service/pkg/lifecycle"
+	"github.com/safeblock-dev/wr/taskgroup"
+)
+
+func main() {
+	ctx := context.Background()
+
+	cfg, err := config.Load[config.Agent]()
+	if err != nil {
+		log.Println("cfg load err: ", err)
+		os.Exit(1)
```

Comment:
Можно заменить на вызов `log.Fatalf("cfg load err: %v", err)` так как это вызов делать примерно то же самое.
```suggestion
                log.Fatalf("cfg load err: %v", err)
```